packages/cli/src/commands/build.ts

```diff
@@ -710,7 +710,9 @@ function expandBuild(files: string[], build: Builder): Builder[] {
     });
   }
 
-  let src = normalize(build.src || '**');
+  let src = normalize(build.src || '**')
+    .split(sep)
+    .join(posix.sep);
```

Comment:
```suggestion
    .join('/');
```

Nit pick: this should be a literal since the following line is going to check for `./`

Reply:
Fixed!